homeassistant/components/manual_mqtt/alarm_control_panel.py

```diff
@@ -446,11 +497,46 @@ async def message_received(msg):
             self.hass, self._command_topic, message_received, self._qos
         )
 
+        config = {
```

Comment:
We don't want to handle config.

Reply:
> We don't want to handle config.

Who is "we"?  @thanksmister already has this implemented -- it makes configuring the MQTT Panel much easier because HA can publish the configuration to the panel.

Again, I am happy to split this out into another PR IF that will be reviewed quickly.

Reply:
We as in Home Assistant core.